04_numeronym/mohitnag/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	ns := []string{}
+	for _, val := range vals {
```

Comment:
size of `ns` is known and could potentially be preallocated

Reply:
Done